Makefile

```diff
@@ -35,4 +38,9 @@ prepare:
 	glide install
 
 run:
-	cd bin && ./edgex-ui-go-launch.sh
\ No newline at end of file
+	cd bin && ./edgex-ui-go-launch.sh
+
+docker: $(DOCKERS)
+
+docker_edgex_ui_go:
+	docker build -f docker-file/Dockerfile -t edgexfoundry/docker-edgex-ui-go:$(VERSION) .
```

Comment:
We also need to add a label flag here that puts in the GIT_SHA.

Reply:
ok, i will fix it

Reply:
Thanks @badboy-huaqiao it should be a really easy fix.
Add a line like this https://github.com/edgexfoundry/edgex-go/blob/master/Makefile#L25
and then add a flag in the docker build command string like this
https://github.com/edgexfoundry/edgex-go/blob/master/Makefile#L81

With this we can always inspect a container and see the exact code it was built from.

Reply:
@JPWKU thanks for your reminding，it's done now ! if any need to be modified, please leave msg here, thanks.